[2022-06-13 10:20:04,478] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_test_iteration.end manual__2022-06-13T10:14:52.630893+00:00 [queued]>
[2022-06-13 10:20:04,500] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: a_test_iteration.end manual__2022-06-13T10:14:52.630893+00:00 [queued]>
[2022-06-13 10:20:04,501] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 10:20:04,502] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-13 10:20:04,502] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-13 10:20:04,526] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): end> on 2022-06-13 10:14:52.630893+00:00
[2022-06-13 10:20:04,551] {standard_task_runner.py:52} INFO - Started process 19008 to run task
[2022-06-13 10:20:04,556] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'a_test_iteration', 'end', 'manual__2022-06-13T10:14:52.630893+00:00', '--job-id', '233', '--raw', '--subdir', 'DAGS_FOLDER/call_dag.py', '--cfg-path', '/tmp/tmpixpux4zd', '--error-file', '/tmp/tmpwjq59eiw']
[2022-06-13 10:20:04,561] {standard_task_runner.py:80} INFO - Job 233: Subtask end
[2022-06-13 10:20:04,675] {task_command.py:370} INFO - Running <TaskInstance: a_test_iteration.end manual__2022-06-13T10:14:52.630893+00:00 [running]> on host 8ef6f3d8e77e
[2022-06-13 10:20:04,770] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=a_test_iteration
AIRFLOW_CTX_TASK_ID=end
AIRFLOW_CTX_EXECUTION_DATE=2022-06-13T10:14:52.630893+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-06-13T10:14:52.630893+00:00
[2022-06-13 10:20:04,772] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-06-13 10:20:04,773] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'echo end']
[2022-06-13 10:20:04,786] {subprocess.py:85} INFO - Output:
[2022-06-13 10:20:04,787] {subprocess.py:92} INFO - end
[2022-06-13 10:20:04,788] {subprocess.py:96} INFO - Command exited with return code 0
[2022-06-13 10:20:04,838] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=a_test_iteration, task_id=end, execution_date=20220613T101452, start_date=20220613T102004, end_date=20220613T102004
[2022-06-13 10:20:04,892] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-13 10:20:04,949] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
